### DISCUSSION TITLE
Generate repos config file from existing repositories

### DISCUSSION THREAD

**Discussion by mpailloncy** [github/safe-settings#666] [Q&A]
Hi all,

Is there already a way to generate Yaml files for a list of existing repositories of an org ?
When setting up safe-setttings on an existing org, what the recommended way to "import" existing repositories ?

Thanks in advance !

**Comment by HagegeR** [comment]
although it seems feasable, I don't think it would be possible to infer the suborgs yamls, deployements, and everything might end up in each repo, which might take you a long time to organize

But I do like the idea a lot
